### PR TITLE
Add PITS Global Data Recovery Services to the list of companies

### PR DESCRIPTION
**Pull request by pheianox**
CI Integration is really good, thank you!

### CODE REVIEW

**Review by edigaryev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -40,8 +40,12 @@ Many more companies are using Tart in their internal setups. Here are a few of t
   <a href="https://transloadit.com/" target=_blank>
     <img src="https://github.com/cirruslabs/tart/raw/main/Resources/Users/Transloadit.png" height="65"/>
   </a>
+  <a href="https://www.pitsdatarecovery.net/" target=_blank>
+    <img src="https://github.com/cirruslabs/tart/assets/77569421/b5a99e54-c59e-48c8-b6ef-411eda894a48" height="65"/>
```
Comment (fkorotkov) [inline README.md:44]: Interesting idea to use assets! Could you please check in the image though? Just for the reason of consistency with the rest of the assets in [`Resources/Users/` folder](https://github.com/cirruslabs/tart/tree/main/Resources/Users).

File: README.md [hunk 2]
```diff
@@ -40,8 +40,12 @@ Many more companies are using Tart in their internal setups. Here are a few of t
...
+  </a>
 </p>
 
+
```
Comment (fkorotkov) [inline README.md:48]: Seems like it is unnecessary change 🤔 

```suggestion
```

**Review by fkorotkov — COMMENTED** [review]
Glad to hear you enjoy Tart! Would be great to hear more details about how you test data recovery with Tart!

**Comment by pheianox** [comment]
Dear @fkorotkov,

Thank you for your interest in our usage of Tart. We are utilizing Tart as part of our development and testing infrastructure to create and manage macOS and Linux virtual machines on Apple Silicon. We find Tart to be a reliable and efficient tool for our needs.

Regarding your question about testing data recovery with Tart, we are primarily using it to test the functionality of our backup and restore processes. By simulating various failure scenarios and recovering data from Tart-managed virtual machines, we can ensure the integrity and reliability of our data recovery procedures.

We hope this information is helpful and please let us know if you have any further questions or concerns.

Best regards,
@pheianox

**Review by fkorotkov — APPROVED** [review]
